src/ui_elements/appearance/appearance_editor.tscn

```diff
@@ -260,7 +260,7 @@ __meta__ = {
 [node name="Label" type="Label" parent="RandomConfirm/MarginContainer/VBoxContainer"]
 margin_right = 782.0
 margin_bottom = 74.0
-text = "Do you wish to discard your appearance?"
+text = "Generate a new appearance?"
```

Comment:
Wouldn't "generate a new random appearance" be better, to emphasize that you're going to make something random instead of what you made manually?

Reply:
Sure